core/common/text/src/main/java/org/eclipse/rdf4j/common/text/StringUtil.java

```diff
@@ -40,7 +40,10 @@
 	 * @param news The String is the new content.
 	 * @param text The String in which the substitution is done.
 	 * @return The result String containing the substitutions; if no substitutions were made, the result is 'text'.
+	 * 
+	 * @deprecated use {@link String#replace(CharSequence, CharSequence) instead.
 	 */
+	@Deprecated
```

Comment:
Perhaps the method could be removed altogether, instead of just being deprecated ?

Reply:
It probably could, but I didn't want to make that decision in this PR.